telegram/ext/filters.py

```diff
@@ -131,6 +151,7 @@ def filter(self, message):
             return True
 
     all = _All()
+    """:obj:`Filter`: All Messages."""
```

Comment:
Changed the type for filter in Messagehandler to `filter` and added a :mod: link to telegram.ext.filters